developer_guide.md

```diff
@@ -143,6 +144,14 @@ Start the flask webserver locally at localhost:8080
 ./run_server.sh
 ```
 
+If you don't have DataCommons GCP permissions, can run
```

Comment:
Should we move the flask section up (before prerequisites) and change it to "simple / lite" instead of not recommended?

Reply:
Done